bin/init-ext.sh

```diff
@@ -33,7 +33,7 @@ install_requirements () {
 }
 
 . ${APP_DIR}/bin/activate
-
+pip install "setuptools>=44.1.0,<71"
```

Comment:
should only run on 2.9 builds only. 
2.10 python3 should not have this issue (or am i mistaken?)

Reply:
It only _needs_ to pin setuptools on 2.9 builds, but it seems to work fine on later versions for now. If it becomes a problem, we can easily adjust it.